src/sdk/USReverseGeoApi/Address.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace SmartyStreets.USReverseGeoApi
+{
+	using System.Runtime.Serialization;
+
+	/// <summary>
+	///     See "https://smartystreets.com/docs/cloud/us-reverse-geo-api#address"
+	/// </summary>
+	[DataContract]
+	public class Address
+	{
+		#region [ Fields ]
+
+		[DataMember(Name = "street")]
+		public string Street { get; set; }
+
+		[DataMember(Name = "city")]
+		public string City { get; set; }
+
+		[DataMember(Name = "state_abbreviation")]
+		public string StateAbbreviation { get; set; }
+
+		[DataMember(Name = "zipcode")]
+		public string Zipcode { get; set; }

```

Comment:
It looks like we've used `ZipCode` in other instances: https://github.com/smartystreets/smartystreets-dotnet-sdk/blob/master/src/sdk/USStreetApi/Components.cs#L56